src/client/components/AddNewRoom/RoomForm.js

```diff
@@ -1,32 +1,53 @@
 import React from 'react';
 import './AddNewRoomStyle.css';
+import ChatGroup from '../AddPeopleToRoom/ChatGroup';
 import PropTypes from 'prop-types';
 
-const RoomForm = (props) => {
-  const { onCreate, inputChange, roomName, roomId } = props;
+const RoomForm = ({
+  onCreate,
+  inputChange,
+  roomName,
+  roomId,
+  displaypeopleInGroup,
+  backToAddPeople,
+}) => {
   if (roomId) {
-    return `welcome to new room with id ${roomId} and add people in Room `;
+    return <ChatGroup roomId={roomId} roomName={roomName} />;
   }
-  if (!roomId) {
-    return (
+  // if (!roomId) {
+  return (
+    <div className="main">
       <form onSubmit={onCreate}>
-        <button type="submit">Add Room</button>
-        <input
-          type="text"
-          placeholder="Add new Room"
-          onChange={inputChange}
-          value={roomName}
-          required
-        />
+        <div className="form-input">
+          <input
+            type="text"
+            placeholder="please write group name here"
+            onChange={inputChange}
+            value={roomName}
+            required
+          />
+          <button type="submit">Create Room</button>
+        </div>
       </form>
-    );
-  }
+      <div className="heading-members-channel">
+        <h2>Members of the group</h2>
+      </div>
+      <div className="container-roomform"> {displaypeopleInGroup()}</div>
+      <div className="back-button">
+        <button type="button" onClick={backToAddPeople}>
+          Back
+        </button>
+      </div>
+    </div>
+  );
 };
 
-RoomForm.PropTypes = {
+RoomForm.propTypes = {
+  onCreate: PropTypes.func.isRequired,
+  inputChange: PropTypes.func.isRequired,
```

Comment:
For consistency, name it `onRoomNameChange` since it is tied to the `roomName` prop.